lib/rubocop/options.rb

```diff
@@ -467,8 +467,14 @@ module OptionsHelp
                                          'This option applies to the previously',
                                          'specified --format, or the default format',
                                          'if no format is specified.'],
-      fail_level:                       ['Minimum severity (A/I/R/C/W/E/F) for exit',
-                                         'with error code.'],
+      fail_level:                       ['Minimum severity for exit with error code.',
+                                         '[A] autocorrect',
+                                         '[I] info',
+                                         '[R] refactor',
+                                         '[C] convention',
+                                         '[W] warning',
+                                         '[E] error',
+                                         '[F] fatal'],
```

Comment:
We already allow both words and abbreviations:
https://github.com/rubocop/rubocop/blob/9f82e0fb2a581283e588a7161839a10c77a2509c/spec/rubocop/options_spec.rb#L298-L301